.nvmrc

```
@@ -0,0 +1 @@
+v14.x
```

```suggestion
v14.16.1
```
You'll want to specify an exact version (I think you can also use release aliases in theory, but having an exact version makes a dev environment more consistent and reproducible, same reason as following the lockfile for deps)

---

fair point, ill do just that.